recipes/bsp/grub2.yaml

```diff
@@ -0,0 +1,84 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+    PKG_VERSION: "2.12"
+
+Config:
+    GRUB2_BOOTSTRAP:
+        help: Run bootstrap in the sources. This is only needed if a git repo is used
+        type: bool
+    GRUB2_TARGET:
```

Comment:
Couldn't we use `$ARCH` for this?

Reply:
Well - we have tests using `i386-pc` legacy boot on a `x86_64` system so I think making it possible to use a different target is not wrong.

Reply:
Makes sense. :+1: